bin/oneoff/merge_studio_persons

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+# This script reads a CSV of data, connecting users via studio_persons using the
+# email address links in the CSV. The CSV is expected to contain at least two
+# columns (later columns are ignored) of emails and alternate emails.
+
+require_relative '../../dashboard/config/environment'
+require 'csv'
+
+FILENAME = ARGV[0].freeze
```

Comment:
Do we expect this script to be used in an ongoing basis, or just one batch? If the former, it should be moved outside of `oneoff`.